src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java

```diff
@@ -1056,4 +1056,16 @@ public void stashChanges(Repository repo) throws GitAPIException {
         git.stashCreate().call();
         git.close();
     }
+
+    /**
+     * Disables auto garbage collection for the given repository.
+     *
+     * @param repository the repository
+     */
+    public void disableAutoGC(org.eclipse.jgit.lib.Repository repository) {
```

Comment:
please move this code into a test service because it will not actively be used on production